advancedDungeon/src/produsAdvanced/level/AdvancedBerryLevel.java

```diff
@@ -69,7 +69,7 @@ public class AdvancedBerryLevel extends BlocklyLevel {
    */
   public AdvancedBerryLevel(
       LevelElement[][] layout, DesignLabel designLabel, List<Coordinate> customPoints) {
-    super(layout, designLabel, customPoints, "Berry");
+    super(layout, designLabel, customPoints, "Berry", "");
```

Comment:
DevDungeonLevel hat eine eigene Logik, muss am Anfang Popups zu erstellen mit dem Levelnamen und einer Beschreibung. Ich glaube das jedes (@AMatutat stimmt das?) AdvancedDungeon Level eine `msg` diese dient ja als Beschreibung. 
```suggestion
    super(layout, designLabel, customPoints, "Berry", msg);
```

Dafür müsste aber in jeden AdvancedLevel die `onFirstTick` angepasst werden, um keine Popups selber zu zeigen.